templates/settingsForm.tpl

```diff
@@ -8,53 +8,64 @@
  * Registration Notification plugin settings
  *
  *}
-<div id="registrationNotificationSettings">
-<div id="description">{translate key="plugins.generic.registrationNotification.description"}</div>
-
-<h3>{translate key="navigation.settings"}</h3>
-
 <script>
 	$(function() {ldelim}
+		function setValidation(container) {ldelim}
```

Comment:
I think it would be better to use an extension of `AjaxFormHandler` that adds this in the handler class itself, rather than building it into the template.

Reply:
I've tried to add the attribute through PHP, but it was blocked by Smarty. I'll take a deeper look at the reason.

Reply:
@asmecher Ah, I thought you were talking about the setValidation function. I've checked and the reason for its existence was really a limitation in Smarty 2.6 (it has a `/^\w+$/` validation for attribute names), but in Smarty 3.* it doesn't exist anymore, so I can remove the workaround in /master.

But now that I've seen some extensions of the AjaxFormHandler, I understood that you meant to move the whole script to an external file.